Index: Classes/Class Pranavi/Agenda.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Day 4: 27th June 2024\r\n----------------------\r\nRecap on Data Type\r\nIdentifiers\r\nRules for Identifiers\r\nBasic DataTypes are Immutable\r\n\r\nDay 7: 02 July 2024\r\n---------------------\r\n1. String Slicing\r\n2. Recap of String Data Type\r\n3. Complex Data Type\r\n4. Float Data Type - Use of Exponent form\r\n\r\nDay 8: 04 July 2024\r\n-------------------------\r\nString Slicing \r\nPython is dynamically typed language\r\nExplained Range DataType\r\nand range in for loop\r\n\r\nQ. difference between byte and character\r\n\r\nQuick Quiz on Basic Data Types and  Type Casting\r\n\r\nDay 9:\r\n---------------------------\r\nAgenda:\r\nQ. What is Type Casting ?\r\nTopic: Quick Recap on all DataTypes:\r\nShow the Excel\r\n\r\n\r\nNotes:\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Class Pranavi/Agenda.txt b/Classes/Class Pranavi/Agenda.txt
--- a/Classes/Class Pranavi/Agenda.txt	(revision 18699e0fa15b788aabc654f535fcc948c06258da)
+++ b/Classes/Class Pranavi/Agenda.txt	(date 1720494960183)
@@ -26,9 +26,20 @@
 Day 9:
 ---------------------------
 Agenda:
-Q. What is Type Casting ?
+Recap:
+Q. What is Type Casting or Type Conversion?
+Is Python statically typed or dynamically typed?
 Topic: Quick Recap on all DataTypes:
 Show the Excel
-
-
+------------------------
 Notes:
+Actual:
+--------
+if else using two programs
+we reviewed type casting
+concept of flag variable
+--------------
+Homework:
+WAPP to check if a given number is even or odd
+WAPP To tell if the given number is positive or negative
+
Index: Classes/Class Pranavi/Day009_09-July-2024/01 Biggest of Two.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Class Pranavi/Day009_09-July-2024/01 Biggest of Two.py b/Classes/Class Pranavi/Day009_09-July-2024/01 Biggest of Two.py
new file mode 100644
--- /dev/null	(date 1720493580566)
+++ b/Classes/Class Pranavi/Day009_09-July-2024/01 Biggest of Two.py	(date 1720493580566)
@@ -0,0 +1,13 @@
+num1 = int(input("enter a number: "))
+num2 = int(input("enter a number: "))
+largest = 0
+
+if num1 > num2:
+    largest = num1
+else:
+    largest = num2
+
+# if num2 > num1:
+#     largest = num2
+
+print(largest)
\ No newline at end of file
Index: Classes/Class Pranavi/Day009_09-July-2024/02 Decide Where to Play.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Class Pranavi/Day009_09-July-2024/02 Decide Where to Play.py b/Classes/Class Pranavi/Day009_09-July-2024/02 Decide Where to Play.py
new file mode 100644
--- /dev/null	(date 1720494960207)
+++ b/Classes/Class Pranavi/Day009_09-July-2024/02 Decide Where to Play.py	(date 1720494960207)
@@ -0,0 +1,23 @@
+# flag variable
+weather = input("Enter weather condition: ")
+
+is_playing_inside = True
+
+if weather == "raining":
+    print("Play Inside")
+    print("Play Inside")
+    print("Play Inside")
+else:
+    is_playing_inside = False
+    print("Play outside")
+    print("Play outside")
+    print("Play outside")
+
+
+print("PrNAVI IS PLAYING")
+
+
+# if weather == "raining":
+#     play_inside = True
+# else:
+#     play
\ No newline at end of file
Index: Classes/siri/Chapter 01/Data Types/0x Bytes and BytesArray/0x String vs BytesArray.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/siri/Chapter 01/Data Types/0x Bytes and BytesArray/0x String vs BytesArray.py b/Classes/siri/Chapter 01/Data Types/0x Bytes and BytesArray/0x String vs BytesArray.py
new file mode 100644
--- /dev/null	(date 1720514224079)
+++ b/Classes/siri/Chapter 01/Data Types/0x Bytes and BytesArray/0x String vs BytesArray.py	(date 1720514224079)
@@ -0,0 +1,17 @@
+ba = bytearray(b'\x00\x01\x02\x03\x04')
+print(ba)  # Output: bytearray(b'\x00\x01\x02\x03\x04')
+
+# Delete the byte at index 1
+del ba[1]
+print(ba)  # Output: bytearray(b'\x00\x02\x03\x04')
+
+# Delete a slice of bytes
+del ba[1:3]
+print(ba)  # Output: bytearray(b'\x00\x04')
+
+ba = bytearray(b'Hello')
+print(ba)
+ba[0:1] = b'Y'
+ba[0] = ord('y')
+print(ba)
+
Index: Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 01.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>a = 10\r\nb = 20\r\nprint(f\"a > b is {a > b}\")\r\nprint(f\"a > b is {a >= b}\")\r\nprint(f\"a > b is {a < b}\")\r\nprint(f\"a > b is {a <= b}\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 01.py b/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 01.py
--- a/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 01.py	(revision 18699e0fa15b788aabc654f535fcc948c06258da)
+++ b/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 01.py	(date 1720495327232)
@@ -1,6 +1,6 @@
-a = 10
+a = 20
 b = 20
-print(f"a > b is {a > b}")
-print(f"a > b is {a >= b}")
-print(f"a > b is {a < b}")
-print(f"a > b is {a <= b}")
\ No newline at end of file
+print(f"{a} > {b} is {a > b}")  # 20 > 20 is False
+print(f"a >= b is {a >= b}")   # True
+print(f"a < b is {a < b}")  # False
+print(f"a <= b is {a <= b}")  # True  less than or equal to
Index: Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 02.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>a = \"Apple\"\r\nb = \"Ball\"\r\nprint(f\"a > b is {a > b}\")\r\nprint(f\"a >= b is {a >= b}\")\r\nprint(f\"a < b is {a < b}\")\r\nprint(f\"a <= b is {a <= b}\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 02.py b/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 02.py
--- a/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 02.py	(revision 18699e0fa15b788aabc654f535fcc948c06258da)
+++ b/Classes/Class Pranavi/Day009_09-July-2024/01 Relational Operators 02.py	(date 1720495678483)
@@ -1,6 +1,8 @@
 a = "Apple"
-b = "Ball"
-print(f"a > b is {a > b}")
-print(f"a >= b is {a >= b}")
-print(f"a < b is {a < b}")
-print(f"a <= b is {a <= b}")
\ No newline at end of file
+# b = "Ball"
+# b = "Apples"
+b = "apple"  # 'A' - 65 'a' -97 this is greater
+print(f"a > b is {a > b}")  # F
+print(f"a >= b is {a >= b}")  # F
+print(f"a < b is {a < b}")  # T
+print(f"a <= b is {a <= b}")  # T
